routes/web.php

```diff
@@ -45,8 +45,9 @@
   Route::post('/subjects', 'SubjectsController@create');
 
   Route::get('/submissions/progress', 'SubmissionsController@progress');  
-  Route::post('/submissions', 'SubmissionsController@store');
+  Route::post('/submissions/store', 'SubmissionsController@store');
```

Comment:
Doesn't look like you've updated the call to this route from `Assignments.js`.
Saying that, I think we should stay consistent and keep this route as `POST /submissions`.
But maybe rename the controller method to be `create` instead of `store` so it's clear that the action is creating a new submission.